custom_components/nhc2/nhccoco/devices/device.py

```diff
@@ -123,7 +125,7 @@ def extract_property_definition(self, property_key: str) -> str:
                 return property_definition_object[property_key]
         return None
 
-    def extract_property_definition_description_choices(self, property_key: str) -> str:
+    def extract_property_definition_description_choices(self, property_key: str) -> Union[List, None]:
```

Comment:
Why this `List` and not `list`. 
I'm not a python expert, and throughout the codebase `list` is used. Can you change this for the sake of having consitency